weave/compile.py

```diff
@@ -173,6 +173,10 @@ def _dispatch_map_fn_no_refine(node: graph.Node) -> typing.Optional[graph.Output
         ):
             output_type = op.unrefined_output_type_for_params(params)
 
+        # Consider: If the UI does a `callOpVeryUnsafe`, it is possible that the
```

Comment:
I don't want to mess with compile/dispatch right now due to the sensitive nature of the area. However, we might want to add this logic in the future